Library/Formula/osquery.rb

```diff
@@ -44,6 +44,7 @@ def install
                                  "--record=installed.txt"}
     end
 
+    mkdir_p "#{buildpath}/generated"
```

Comment:
Also: why is this needed? Seems like something the CMake buildsystem should handle.